test/Utils/src/org/openj9/test/util/IncludeExcludeTestAnnotationTransformer.java

```diff
@@ -43,6 +43,11 @@
 		String excludeFile = System.getenv("EXCLUDE_FILE");
 		logger.info("exclude file is " + excludeFile);
 		try {
+			if (excludeFile == null || excludeFile.length() == 0) {
+				File temp = File.createTempFile("exclude", ".txt");
+				temp.deleteOnExit();
+				excludeFile = temp.getAbsolutePath();
+			}
```

Comment:
Do we have to create an empty file? Is it possible to skip read file code completely if exclude file does not exist?
```
if (excludeFile != null || excludeFile.length() > 0) {
   // read file
  try {
      FileReader fileReader = new FileReader(excludeFile);
      ...
}
```

Reply:
It does not work to skip the code completely here (I tried).  I will look to see where the testng framework loads the transformer, to see if we have an option there to take action of ignoring the transformation step entirely.

Reply:
This is expected as error 'Cannot instantiate... IncludeExcludeTestAnnotationTransformer' was not triggered by FileReader. The null file has been caught by "FileNotFoundException".

Reply:
Actually I tried with  if (excludeFile!=null) { } it works. The error 'Cannot instantiate...' is caused by runtime FileReader(null), which can not be caught by exception.  If directly trying FileReader(null), you will get compile error.  FileNotFoundException only works for filename or directory is wrong, which doesn't include this null situation.

Reply:
Yes, that was my original approach, but when I 'tested' locally did not seem to work...  Perhaps a late night attempt that I did not do properly.  I have tried again, and pushed the updated change.